module_1/src/com/epam/cycles/CyclesTaskB.java

```diff
@@ -0,0 +1,33 @@
+package com.epam.cycles;
+import java.util.Scanner;
+
+public class CyclesTaskB {
+
+    private static final double CONDITION_PARAMETER = 2.0;
+
+    public static void main(String[] args) {
+
+        Scanner console = new Scanner(System.in);
+        System.out.println ("Set the interval [a,b]:");
+        System.out.print (" -- enter the value 'a': ");
+        double a = console.nextDouble();
+        System.out.print (" -- enter the value 'b': ");
+        double b = console.nextDouble();
+        System.out.print ("Set the step value (h): ");
```

Comment:
Still spaces after System.out.println
```suggestion
        System.out.println("Set the interval [a,b]:");
        System.out.print(" -- enter the value 'a': ");
        double a = console.nextDouble();
        System.out.print(" -- enter the value 'b': ");
        double b = console.nextDouble();
        System.out.print("Set the step value (h): ");
```